src/nvapi_d3d.cpp

```diff
@@ -272,7 +272,16 @@ extern "C" {
         if (nvapiD3dInstance->IsUsingLfx() || !NvapiD3dLowLatencyDevice::SupportsLowLatency(pDev))
             return NoImplementation(n, alreadyLoggedNoImpl);
 
-        if (!NvapiD3dLowLatencyDevice::SetLatencyMarker(pDev, pSetLatencyMarkerParams->frameID, pSetLatencyMarkerParams->markerType))
+        auto markerType = NvapiD3dLowLatencyDevice::ToMarkerType(pSetLatencyMarkerParams->markerType);
```

Comment:
Same thing should be done in `NvAPI_D3D12_SetAsyncFrameMarker`.

Reply:
ah, should practice to not only search in the current file ;)